build-packages/webpack-i18n-runtime/build-config/config.plugin.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-param-reassign */
+const path = require('path');
+const WebpackI18nTracker = require('./webpack-i18n-tracker');
+
+// Provide an additional loader directory from this module
+function addResolveLoader(config) {
+    if (!config.resolveLoader) {
+        config.resolveLoader = {};
+    }
+    if (!config.resolveLoader.modules) {
+        config.resolveLoader.modules = [];
+    }
+    if (!config.resolveLoader.modules.includes('node_modules')) {
+        config.resolveLoader.modules.push('node_modules');
+    }
+    config.resolveLoader.modules.push(__dirname);
+}
+
+// Load the locale map with import injector
+function addImportInjector(config) {
+    config.module.rules.push({
+        test: require.resolve(path.join(__dirname, '../src/util/localeMap.js')),
+        loader: 'webpack-i18n-import-loader'
+    });
+}
+
+// Provide the __ function with the ProvidePlugin
+function provideTranslationFunction(config) {
+    config.plugins.forEach(((plugin) => {
+        // if (plugin instanceof webpack.ProvidePlugin) {
+        const { __proto__: { constructor: { name } = {} } = {} } = plugin;
+        if (name === 'ProvidePlugin') {
```

Comment:
Can you please retry with instance of?

Reply:
Just to check, validate if there is only single one installed (you can use yarn.lock)

Reply:
It works now :0